.appveyor.yml

```diff
@@ -1,20 +1,39 @@
-image:
-- Visual Studio 2019
-platform:
-- x64
-configuration:
-- Release
-cache:
-- C:\Tools\vcpkg
+skip_commits:
+  files:
+    - '*travis*'
+    - '*.md'
+    - '.github/**'
+    - '.clang-format'
+    - 'CHANGES'
+    - 'LICENSE'
+
+skip_branch_with_pr: true
```

Comment:
What does this do and what is the justification for it?

Reply:
This option prevents AppVeyor from triggering two builds for a branch with an open PR.